automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/parameterSettings.yml

```diff
@@ -249,6 +271,82 @@ rules:
     pattern: |
       (@variable:Identifier | @variable:Phrase) [word="of"] @value:Value (?=@unit:Unit)
 
+  - name: var_stay_around_to
+    label: IntervalParameterSetting
+    priority: ${priority}
+    type: dependency
+    action: ${paramIntAction}
+    example: "most of the daily new number of confirmed cases from the community stayed around 15 to 25"
+    pattern: |
+      trigger = [lemma='stay']
+      variable:Phrase = nsubj
+      valueLeastIncl:Value = dobj compound
+      valueMostIncl:Value = dobj
+
+  - name: var-assigned
+    label: ParameterSetting
+    priority: ${priority}
+    type: dependency
+    example: "the curve number is assigned a value of 98 regardless of the soil water content."
+    action: ${paramSetAction}
+    pattern: |
+      trigger = [lemma=/assign|assume/]
+      variable:Phrase = nsubjpass
+      value:Value = dobj nmod_of | xcomp nummod
+
+
+  - name: var_lower_limit
+    label: IntervalParameterSetting
+    priority: ${priority}
+    type: dependency
+    action: ${paramIntAction}
+    example: ""
+    pattern: |
+      trigger = (?<="lower") [word="limit"]
+      variable:Phrase = nmod_of
+      valueLeastIncl:Value = <nsubj
+
+  - name: var-constant
+    label: ParameterSetting
+    priority: ${priority}
+    type: dependency
+    example: "The labile P immobilization factor varies linearly from 0.01 for cLP = 0 to 0.02 for cLP = 10 and remains constant at 0.02 for cLP > 10."
+    action: ${paramSetAction}
+    pattern: |
+      trigger = [lemma="constant"]
+      variable:Phrase = <xcomp nsubj
+      value:Value = <xcomp nmod_at
+
+  - name: value_var
+    label: ParameterSetting
+    priority: ${priority}
+    type: token
+    example: "At planting time, the model takes a soil sample and applies up to 15 kg/ha of N fertilizer if needed."
+    action: ${paramSetAction}
+    pattern: |
+      @value:Value [word=/\w*/]{,4} of @variable:Phrase
+
+  - name: value_of
+    label: ParameterSetting
+    priority: ${priority}
+    type: dependency
+    example: "If the parameters above are used to simulate the future spread of epidemic we obtain the value of R∞ to be 350."
+    action: ${paramSetAction}
+    pattern: |
+      trigger = [lemma="value"]
+      variable:Phrase = nmod_of | compound | <dobj nsubj
+      value:Value = <dobj advcl_to | nmod_of | <nsubj | <nsubjpass xcomp nummod
```

Comment:
I'm not sure if this is an error at my end, but when I run the above example, "we" is extracted as a variable instead of R∞. Could you check that for me? (I'm trying at your moreParamSettings branch.)

Reply:
fixed; added a test.